keps/sig-node/4816-dra-prioritized-list/README.md

```diff
@@ -0,0 +1,842 @@
+notes and implementing the feature itself. KEP editors and SIG Docs
+should help to ensure that the tone and content of the `Summary` section is
+useful for a wide audience.
+
+A good summary is probably at least a paragraph in length.
+
+Both in this section and below, follow the guidelines of the [documentation
+style guide]. In particular, wrap lines to a reasonable length, to make it
+easier for reviewers to cite specific portions, and to minimize diff churn on
+updates.
+
+[documentation style guide]: https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md
+-->
+
+
+The [DRA Structured
+Parameters](https://git.k8s.io/enhancements/keps/sig-node/4381-dra-structured-parameters)
+feature has added the ability to make requests for very specific types of
+devices using a `ResourceClaim`. However, the current API does not allow the
+user to indicate any priority when multiple types or configurations of devices
+may meet the needs of the workload. This feature allows the user to specify
+alternative requests that statisfy the workloads need, giving the scheduler more
+flexiblity in scheduling the workload.
+
+## Motivation
+
+<!--
+This section is for explicitly listing the motivation, goals, and non-goals of
+this KEP.  Describe why the change is important and the benefits to users. The
+motivation section can optionally provide links to [experience reports] to
+demonstrate the interest in a KEP within the wider Kubernetes community.
+
+[experience reports]: https://github.com/golang/go/wiki/ExperienceReports
+-->
+
+"Obtainability" of certain types of scarce resources is a primary concern of
+many AI/ML users. GPUs are in high demand, particularly the latest models. This
+means that workloads that use DRA to specify a need for particular types of GPUs
+may fail to schedule. In practice, a workload that needs a GPU can be written
+such that it can discover the GPUs available to it, and work with what it is
+given. A user may have a preference for the latest model, but would like to run
+the workload even if only an older model is available.
+
+Similarly, packaged workload authors may wish to configure a workload such that
+it will work well in the widest selection of available clusters. That is, a
+distributor of shared workload definitions would like to be able to specify
+alternative types of devices with which their workload will function, without
+requiring the user to modify the manifests.
+
+### Goals
+
+<!--
+List the specific goals of the KEP. What is it trying to achieve? How will we
+know that this has succeeded?
+-->
+
+* Allow workload authors, when specifying a `ResourceClaim`, to provide a list
+  of ways to satisfy the claim, with a preference ranking.
+* Enable the scheduler to evaluate those preferences and allocate devices for the
+  claim based on them.
+* Enable the cluster autoscaler to evaluate those preferences and make scaling
+  choices based on them.
+
```

Comment:
Will Kueue and/or Resource Quotas be in scope for this?

Reply:
Added something about quota, lmk what you think